src/qt/walletmodel.cpp

```diff
@@ -482,7 +482,7 @@ WalletModel::UnlockContext::~UnlockContext()
     }
 }
 
-void WalletModel::UnlockContext::CopyFrom(const UnlockContext& rhs)
+void WalletModel::UnlockContext::CopyFrom(UnlockContext&& rhs)
 {
     // Transfer context; old object no longer relocks wallet
     *this = rhs;
```

Comment:
Is this calling `UnlockContext& operator=(UnlockContext&& rhs)`?

Reply:
No, this calls the (private) default copy constructor.

Reply:
Because the rvalue arg becomes an lvalue once it's named (at the beginning of the function).